core/instance.go

```diff
@@ -293,8 +293,8 @@ func (i *instance) getCheapestCompatibleSpotInstanceType(allowedList []string, d
 
 	for _, candidate := range i.region.instanceTypeInformation {
 
-		logger.Println("Comparing ", candidate.instanceType, " with ",
-			current.instanceType)
+		logger.Printf("Comparing %s with %s ",
+			current.instanceType, candidate.instanceType)
```

Comment:
Why switching this one to `Printf` but leaving the other as `Println` ?

Reply:
I had a larger local patch in which I wanted to print on the same line the reason why the comparison failed: CPU, memory, storage, etc.

I failed to do that cleanly for now, so I reverted most of that but this hunk was missed.